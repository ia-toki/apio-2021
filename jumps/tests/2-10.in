209636825ecd2c38299498cd26890ef71dd128c2
200 200
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200
0 0 199 199
19 70 185 195
29 53 97 97
6 39 46 177
7 24 196 199
67 160 164 195
13 16 83 147
11 166 179 180
159 159 192 197
7 149 155 175
125 146 195 198
4 8 102 173
27 130 167 171
71 99 104 154
1 65 154 171
5 16 134 170
2 7 139 154
26 32 69 165
3 53 81 188
12 68 80 139
56 110 124 124
88 181 193 197
25 29 196 198
20 23 46 129
46 57 97 168
0 3 98 160
11 28 142 148
152 179 185 189
76 87 110 121
91 163 198 198
1 10 126 143
7 17 186 188
104 136 164 199
80 174 181 185
15 99 103 178
7 101 168 182
18 47 175 195
11 11 153 185
73 80 151 170
127 154 156 168
36 150 181 185
130 156 178 182
64 105 113 198
111 138 165 176
92 184 191 193
123 148 156 157
58 168 170 181
60 60 198 199
16 21 161 175
0 73 105 123
26 30 58 152
13 15 80 156
78 173 199 199
135 162 199 199
121 198 199 199
134 176 195 198
93 93 100 102
49 79 190 190
2 7 99 117
4 11 65 143
110 181 187 193
0 2 194 198
89 95 129 137
8 55 87 185
25 70 120 188
32 86 150 187
80 96 196 196
97 103 185 190
97 107 194 196
3 9 65 190
27 51 79 186
115 120 193 199
4 15 43 82
5 55 96 167
6 31 101 169
21 43 85 100
79 132 151 179
85 190 196 197
78 122 173 188
57 57 138 180
121 163 192 199
144 164 189 198
1 99 144 151
20 116 139 148
2 5 177 186
46 52 147 193
84 139 196 196
20 120 143 144
19 58 141 164
0 57 156 192
33 67 182 195
125 195 199 199
0 8 162 189
96 190 192 195
14 99 168 195
72 105 130 170
64 82 103 168
177 179 192 198
127 161 182 184
89 143 173 192
6 9 51 103
126 179 195 196
96 172 197 199
43 135 167 173
90 190 192 197
1 32 136 196
123 177 186 187
108 182 183 197
23 65 155 183
6 89 174 188
6 7 135 180
17 87 105 119
1 39 83 151
15 30 176 179
0 127 151 182
54 74 181 196
92 115 126 167
10 189 197 199
102 198 199 199
10 10 25 99
84 84 145 199
94 172 195 196
98 124 157 191
69 111 119 181
154 176 180 188
10 10 46 89
21 112 118 131
65 105 197 197
28 62 170 173
33 86 121 188
1 103 116 190
62 68 174 194
53 78 105 191
39 125 147 178
10 36 116 137
41 194 197 199
63 152 172 181
51 101 157 188
15 29 40 50
41 44 70 100
65 143 146 163
25 25 122 178
46 145 194 194
5 61 71 71
47 82 136 169
45 150 160 189
1 1 63 130
71 169 189 199
27 47 70 130
82 187 192 198
38 91 94 135
25 60 186 195
10 81 127 156
4 113 147 181
23 86 145 155
15 109 167 197
9 15 93 118
89 193 194 198
38 132 170 188
16 99 137 145
91 147 194 194
66 138 199 199
3 16 82 194
20 26 116 119
4 6 95 108
139 180 182 191
75 146 150 172
22 30 157 193
56 57 195 199
12 23 188 192
102 149 162 162
23 96 189 194
55 136 170 188
63 119 137 143
151 198 199 199
7 39 171 197
8 14 132 189
96 112 157 164
68 145 160 179
97 196 199 199
33 33 72 96
69 84 93 133
109 118 186 197
29 81 96 132
5 54 103 173
15 94 114 135
24 48 198 199
3 25 103 169
29 180 191 197
7 19 180 195
36 44 162 179
123 137 180 186
37 66 75 167
18 29 137 166
35 51 149 157
34 121 185 195
4 9 19 169
16 28 193 197
72 140 171 189
120 171 186 191
