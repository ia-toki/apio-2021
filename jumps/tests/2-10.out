fc5005cd102fffc504bc1615010ebf88823d65f3
OK
199
115
44
7
172
4
67
13
33
6
49
94
37
5
89
118
132
37
28
12
14
12
167
23
40
95
114
6
23
35
116
169
28
7
4
67
128
142
71
2
31
22
8
27
7
8
2
138
140
32
28
65
26
37
1
19
7
111
92
54
6
192
34
32
50
64
100
82
87
56
28
73
28
41
70
42
19
6
51
81
29
25
45
23
172
95
57
23
83
99
115
4
154
2
69
25
21
13
21
30
42
16
25
32
2
104
9
1
90
85
128
18
44
146
24
107
11
8
1
15
61
23
33
8
4
36
6
92
108
35
13
106
27
22
80
3
20
56
11
26
3
97
49
10
54
10
62
20
23
5
3
126
46
34
59
58
78
1
38
38
47
61
66
90
89
2
4
127
138
165
13
93
34
18
1
132
118
45
15
3
39
9
68
15
49
20
150
78
11
161
118
43
9
108
98
64
10
165
31
15
